packages/models/mock/Organization/avengers.json

```
@@ -0,0 +1,24 @@
+{
+  "id": "avengers",
+  "type": "Organization",
+  "attributes": {
+    "name": "Avengers",
+    "additionalType": "PerformingGroup"
+  },
+  "relations": {
+    "members": [
```

Use predicate semantics, ignore plural

```suggestion
    "hasMember": [
```

> Inverse should be `isMemberOf`

---

Use natural language form

```
Alice hasFriend Bob
Alice isFriendOf Bob
```